addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua

```diff
@@ -283,37 +283,44 @@ function TOOL:Think()
   self:UpdateGhostWirejoystick( self.GhostEntity, self:GetOwner() )
 end
 
+local function trimTextEntry(pnText, sConvar, nLen)
+  local sTxt = pnSelf:GetText():Trim()
```

Comment:
`pnSelf` is passed into this function but named `pnText` - how does this work?

Reply:
Thanks ... Copied from the control panel. Fixed .